application/src/main/java/org/thingsboard/server/actors/tenant/TenantActor.java

```diff
@@ -84,6 +85,8 @@ public void init(TbActorCtx ctx) throws TbActorException {
                 cantFindTenant = true;
                 log.info("[{}] Started tenant actor for missing tenant.", tenantId);
             } else {
+                systemContext.schedulePeriodicMsgWithDelay(ctx, SessionTimeoutCheckMsg.instance(), systemContext.getSessionReportTimeout(), systemContext.getSessionReportTimeout());
```

Comment:
I can't find where is the scheduled task going down by the end of the lifetime of the particular TenantActor. For example, repartition event